sample/src/androidTest/java/com/schibsted/spain/barista/sample/RecyclerViewTest.java

```diff
@@ -161,4 +161,12 @@ public void checkClickNoButton_byPosition_atSixty_byText() {
     assertDisplayed("'no' has been clicked");
   }
   //endregion
+
+  //region Item count
+  @Test
+  public void checkRecyclerViewItemCount_byDataSetCount() {
+    int expectedCount = RecyclerViewActivity.DATA_COUNT;
+    checkRecyclerViewItemCount(R.id.recycler, expectedCount);
```

Comment:
Let's change the name of this method to `assertItemCount()`. It will meet the style of the other assertions. If not, we will have assertions called `assertSomething` and another ones with `checkSomething`. Let's have only one style.